ajax/libs/query-string/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "query-string",
+  "filename": "index.min.js",
+  "description": "Parse and stringify URL query strings",
+  "version": "5.0.1",
+  "keywords": [
+    "browser",
+    "querystring",
+    "query",
+    "string",
+    "qs",
+    "param",
+    "parameter",
+    "url",
+    "uri",
+    "parse",
+    "stringify",
+    "encode",
+    "decode"
+  ],
+  "author": "Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SaneMethod/sindresorhus/query-string.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/SaneMethod/sindresorhus/query-string.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "*.(js|map)"
```

Comment:
This pattern can't get any files.
Would you please revise this after we got the answer from upstream (which file can be used in web front-end)?
Many thanks

Reply:
@sufuf3 What do you mean by get answer from upstream? There is no way to ask author about file using cause there is not issues tab in lib git repo.
@sindresorhus As I saw you are the author of this lib. Could you help us to know which files can be used in web frontend?